llvm/test/Transforms/LowerConstantIntrinsics/builtin-object-size-range.ll

```diff
@@ -106,4 +107,23 @@ define i64 @select_gep_offsets(i1 %cond) {
   ret i64 %res
 }
 
+define i64 @select_gep_oob_offsets(i1 %cond) {
```

Comment:
Is this actually guaranteed to be OOB? If not,p lease update the test and ideally add a brief comment explaining what this is testing